src/popup/index.tsx

```diff
@@ -151,6 +152,17 @@ export function Popup() {
                 }}
               />
             </FormItem>
+            <FormItem required name='autoTranslate' label='Auto Translate'>
+              <Input
```

Comment:
We can use `Checkbox` component to handle this `boolean` type value

https://baseweb.design/components/checkbox/

Reply:
Thanks for your userful guidance, I have done this, Please review again